parlai/tasks/coqa/build.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+# Download and build the data if it does not exist.
+
+import parlai.core.build_data as build_data
+import os
+import json
+
+tfname = 'coqa-train-v1.0.json'
```

Comment:
constants in python have the convention of being IN_ALL_CAPS_UNDERSCORE.

Additionally we use the convention of "valid" instead of "dev" in ParlAI.

TRAIN_FILENAME
VALID_FILENAME
URL